storage/plugins/archives/RicJsonLdBuilder.php

```
@@ -0,0 +1,1377 @@
+        if (preg_match('#^ark:/#i', $normalised) !== 1) {
+            // Bare NAAN/Name form — must look like `<5+ digits>/<rest>`.
+            if (preg_match('#^[0-9]{5,}/#', $normalised) !== 1) {
+                return null;
+            }
+            $normalised = 'ark:/' . $normalised;
+        } else {
+            // Already prefixed — verify the NAAN segment is digits.
+            if (preg_match('#^ark:/[0-9]{5,}/#i', $normalised) !== 1) {
+                return null;
+            }
+        }
```

PHPMD (2.15.0)

[error] The method sanitizeArkIdentifier uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)